src/Umbraco.Web.UI.Client/src/views/documenttypes/create.html

```diff
@@ -5,35 +5,35 @@ <h5><localize key="create_createUnder">Create an item under</localize> {{current
 
             <ul class="umb-actions umb-actions-child">
                 <li data-element="action-documentType" class="umb-action" ng-hide="model.disableTemplates">
-                    <button type="button" ng-click="createDocType()" class="umb-action-link umb-outline btn-reset" umb-auto-focus>
-                        <umb-icon icon="icon-document" class="icon large"></umb-icon>
+                    <button type="button" ng-click="createDocType('icon-document', 'color-blue')" class="umb-action-link umb-outline btn-reset" umb-auto-focus>
+                        <umb-icon icon="icon-document color-blue" class="icon large"></umb-icon>
```

Comment:
@soreng with the recent changed it should be sufficient with icon name in `icon` attribute. The color class can be added to `class` attribute and should be inherited down to the font icon or SVG icon.

Reply:
@bjarnef I have move the "color-"-classes to the class attribute